downtime/schema.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.schemas.openapi import SchemaGenerator
+
+class DowntimeSchemaGenerator(SchemaGenerator):
+    def get_schema(self, *args, **kwargs):
+        schema = super().get_schema(*args, **kwargs)
+        schema['info']['title'] = 'Downtime Database'
+        schema['info']['description'] = 'An application with a database that stores periods of scheduled telescope downtime for an observatory with an API to access those downtimes.'
+        schema['info']['version'] = 'latest'
```

Comment:
It might be nice to pull this from the git tag. I think there is a way to do that in code like how the setup.py files must figure it out in the observation_portal project...

Reply:
Yeah, good call. From my reading `setuptools_scm` handles this, and it looks like they [shell out to the system's git executable](https://github.com/pypa/setuptools_scm/blob/abb67b15985f380d8cf4451b9f2ef3dd11cb8a91/src/setuptools_scm/git.py#L13), which I'd like to avoid doing, at least explicitly.

I'll have to see if they have a nice higher-level function that just provides the git tag.

Reply:
setuptools_scm provides a handy get_version() function that does exactly this. Added it and it's now properly displaying the latest tag in the docs. https://mgdaily.github.io/ocs-gh-pages/docs/downtime.html

Reply:
Awesome! we can even think about storing the openapi doc by version to have access to the history of the api as well then